subsys/net/lib/lwm2m/lwm2m_senml_cbor_decode.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * Generated using zcbor version 0.3.99
+ * https://github.com/NordicSemiconductor/zcbor
+ * Generated with a --default-max-qty of 99
```

Comment:
Shouldn't this mention `https://github.com/zephyrproject-rtos/zcbor` instead? The same goes for other generated files.